SourceryRuntime/Sources/Enum.swift

```diff
@@ -111,7 +111,7 @@ import Foundation
             self.rawValue = aDecoder.decode(forKey: "rawValue")
             guard let associatedValues: [AssociatedValue] = aDecoder.decode(forKey: "associatedValues") else { NSException.raise(NSExceptionName.parseErrorException, format: "Key '%@' not found.", arguments: getVaList(["associatedValues"])); fatalError() }; self.associatedValues = associatedValues
             guard let annotations: [String: NSObject] = aDecoder.decode(forKey: "annotations") else { NSException.raise(NSExceptionName.parseErrorException, format: "Key '%@' not found.", arguments: getVaList(["annotations"])); fatalError() }; self.annotations = annotations
-            guard let indirect: Bool = aDecoder.decode(forKey: "indirect") else { NSException.raise(NSExceptionName.parseErrorException, format: "Key '%@' not found.", arguments: getVaList(["indirect"])); fatalError() }; self.indirect = indirect
+            self.indirect = aDecoder.decode(forKey: "indirect")
```

Comment:
This change occurred when I ran `rake generate_internal_boilerplate_code` — maybe #813 didn't use this command and instead made this change by hand cc @jacobdweightman ?

Reply:
Yes, my mistake. I couldn't figure out how to make it regenerate this code, so I updated it by hand and assumed I wrote it correctly when CI passed. I'll keep this in mind and remember to run that `rake` command when contributing in the future, though perhaps it would be good to document that this should be done and/or make it a CI requirement.

I'd be more than happy to open up a follow-up PR to make this change—what do you think?